includes/class-fr-thumbnails-folder-image-sizes.php

```
@@ -104,7 +104,7 @@ public function maybe_generate_intermediate_image($downsize, $id, $size) {
 	 *                              or 'update' when the metadata was updated.
      * @return array
 	 */
-    public function maybe_convert_to_image($metadata, $attachment_id, $context) {
+    public function maybe_convert_to_image($metadata, $attachment_id, $context='create') {
```

I understand your intention to address compatibility with other plugins, but I believe it's the responsibility of the code that triggers the filter to provide all necessary parameters. Adding default values can mask issues and may create unintended behavior.